CHANGELOG.md

```diff
@@ -1,5 +1,52 @@
+* Starting on January 1, 2025, [invalid values sent to the Azure AKS API for the properties.mode field of AKS AgentPools will be rejected](https://github.com/Azure/AKS/issues/4468). Prior to this change, unknown modes were assumed to be User. The only valid values for this field are the (case-sensitive) strings:["User", "System"](https://learn.microsoft.com/rest/api/aks/agent-pools/create-or-update?view=rest-aks-2024-02-01&tabs=HTTP#agentpoolmode), or ["Gateway"](https://learn.microsoft.com/rest/api/aks/agent-pools/create-or-update?view=rest-aks-2024-06-02-preview&tabs=HTTP#agentpoolmode).
+
+### Release Notes
+
+* Features:
+  * [Trusted launch](https://learn.microsoft.com/azure/aks/use-trusted-launch), which improves the security of generation 2 virtual machines (VMs) by protecting against advanced and persistent attack techniques is now generally available.
+  * AKS patch versions `1.30.7`, `1.29.11`, `1.28.13`, `1.27.21`, `1.26.22`, and `1.25.24` are now available. Refer to [version support policy](https://learn.microsoft.com/azure/aks/supported-kubernetes-versions?tabs=azure-cli#kubernetes-version-support-policy) and [upgrading a cluster](https://learn.microsoft.com/azure/aks/upgrade-aks-cluster?tabs=azure-cli) for more information..
+
+* Preview features:
+  * AKS version [`1.31`](https://kubernetes.io/blog/2024/08/13/kubernetes-v1-31-release/) is now available in preview. 
+  * You can now specify the default Seccomp profile applied to all workloads by setting the `seccompDefault` on [customized node configuration](https://learn.microsoft.com/azure/aks/custom-node-configuration?tabs=linux-node-pools#linux-kubelet-custom-configuration).  value of `RuntimeDefault` will apply the default from the container's runtime which restricts certain system calls for improved security.
+  * You can now [specify the GPU driver type (preview)](https://learn.microsoft.com/azure/aks/use-windows-gpu) when creating a new AKS Windows GPU Nodepool using the `--driver-type` flag.
+    
+* Bug fixes:
+  * Bug fix to address an issue where Calico pods were stuck in Terminating state.
+  * Fixed a race condition when editing a NetPol or deleting then re-adding it for netpols without CIDR handle.
+  * Fixed a race condition between Cilium and Retina CRDs for Cilium (when Retina is updating to Cilium).
+ 
+* Behavior change:
+  * Deprecated API detection will now only show usage on non-readonly verbs.
+  * Starting with AKS version 1.31, nodes will now [pull container images in a parallel](https://learn.microsoft.com/troubleshoot/azure/azure-kubernetes/availability-performance/container-image-pull-performance) by default. In versions prior to 1.31, the pull type will remain serialized.
```

Comment:
This doc intro is confusing - his change is intended to improve container image pull performance, especially when you deal with large or numerous container images. **However, you might still experience decreased performance compared to using serialized image pulls.**  Why decreased performance ? @allyford

Reply:
@stl327 see comment above

Reply:
This change should improve the experience for nearly all AKS customers. However, it is possible for it to technically regress the experience if the customer is using extremely large container images (multi-GB) and trying to pull many of these images parallelized with an underpowered disk. This would cause disk throttling and could slow the time to finish pulling any of the individual images. But, in our testing which took the production scenario at P99 and worsened it by 5x, parallel pulling still finished pulling all of the images before serial. Some customers may care about finishing the pull of a specific image and if they run into this situation, we created the TSG above to help (or if serial is not performant enough).

Reply:
can we clarify that better in the doc please. @stl327